packages/react-component-library/src/components/SelectE/partials/StyledInlineButtons.tsx

```diff
@@ -1,15 +1,7 @@
 import styled from 'styled-components'
 
-import { StyledInlineButton } from './StyledInlineButton'
+import { StyledInlineButtons as StyledInlineButtonsBase } from '../../../styled-components/partials/StyledInlineButtons'
 
-export const StyledInlineButtons = styled.div`
-  position: relative;
-  display: flex;
-  justify-content: center;
-  text-align: center;
-  align-items: center;
-
-  ${StyledInlineButton} + ${StyledInlineButton} {
-    margin-left: 0;
-  }
+export const StyledInlineButtons = styled(StyledInlineButtonsBase)`
+  border-left: unset;
```

Comment:
`unset` doesn't work with IE unfortunately – so may have to give this an explicit value.